TasmotaSerial-2.4.0 does not work with core 2.3.0

Here is the error message during compilation:

> lib/TasmotaSerial-2.4.0/src/TasmotaSerial.cpp: In member function 'void TasmotaSerial::rxRead()':
> lib/TasmotaSerial-2.4.0/src/TasmotaSerial.cpp:280:7: error: 'TM_SERIAL_WAIT_RCV_LOOP' was not declared in this scope
>        TM_SERIAL_WAIT_RCV_LOOP;    // wait for stop bit
>        ^
> *** [.pioenvs/sonoff/lib879/TasmotaSerial-2.4.0/TasmotaSerial.cpp.o] Error 1

Adding a define line between line 220 and 221 in file TasmotaSerial.cpp fixed it.
#define TM_SERIAL_WAIT_RCV_LOOP { while (ESP.getCycleCount() < (wait + start)); }

--- a/lib/TasmotaSerial-2.4.0/src/TasmotaSerial.cpp
+++ b/lib/TasmotaSerial-2.4.0/src/TasmotaSerial.cpp
@@ -218,6 +218,7 @@ int TasmotaSerial::available()
 #else
 #define TM_SERIAL_WAIT_SND { while (ESP.getCycleCount() < (wait + start)); wait += m_bit_time; }
 #define TM_SERIAL_WAIT_RCV { while (ESP.getCycleCount() < (wait + start)); wait += m_bit_time; }
+#define TM_SERIAL_WAIT_RCV_LOOP { while (ESP.getCycleCount() < (wait + start)); }
 #endif
 
 size_t TasmotaSerial::write(uint8_t b)

